WooCommerce/Classes/ViewModels/ManualTrackingViewModel.swift

```diff
@@ -231,6 +238,12 @@ final class AddCustomTrackingViewModel: ManualTrackingViewModel {
     }
 
     let providerCellAccessoryType = UITableViewCell.AccessoryType.none
+    var isEmptyState: Bool {
```

Comment:
We have this code duplicated above, is there a way to avoid duplicating this? Perhaps through a protocol extension? some utility class might do this work?